pkg/capabilities/pb/registry.proto

```diff
@@ -14,11 +14,16 @@ message RemoteTriggerConfig {
   google.protobuf.Duration messageExpiry = 4;
 }
 
+message RemoteTargetConfig {
+  repeated string requestHashExcludedAttributes = 1;
```

Comment:
how is this supposed to be used? document please.

moreover, why is it a parameter? does the user need to remember to set it on all calls?

why is it an exclusion list rather than inclusion list? the former implies that the we have have not determinism by default, which seems wrong

Reply:
its set in the registry, so config, not per call.  Its true that there is not determinisim by default if you hash the request message due to the behaviour of libocr callbacks where the signature set is different per node, this is expected.  Its exclusion rather than inclusion as  any new added attributes should be hashed by default, it was felt that the exception to that will be rare and explicitly declared as exclude to avoid accidentally not including new attributes on the request.

I'll add some doc.

Reply:
doc added

Reply:
thank you